app/models/miq_region.rb

```diff
@@ -144,13 +139,24 @@ def self.global_replication_type?
   end
 
   def self.replication_type
-    MiqPglogical.new.replication_type
+    if remote_replication_type?
+      :remote
+    elsif global_replication_type?
+      :global
+    else
+      :none
+    end
```

Comment:
Can you rebase?  This was moved in https://github.com/ManageIQ/manageiq/pull/21372, but this looks like it's putting it back?

Reply:
Oh, I did a `git checkout remote-file-depot -- app/models/miq_region.rb` from the previous PR ( https://github.com/ManageIQ/manageiq/pull/21372 ) so I didn't miss anything, but my guess is that PR is out of date from `master` so hence this change.  Will fix!  Good catch!